.gitignore

```diff
@@ -12,3 +12,5 @@
 /man/actionlint.1.html
 /playground-dist
 /actionlint-workflow-ast
+# IntelliJ IDE Folder
+.idea/
```

Comment:
I'll let the maintainers chime in, but typically I'd put this sort of thing in my own global gitignore; people develop in all kinds of environments, so adding that kind of stuff to a project-level gitignore can make it unwieldy